util/encrypt/aes_test.go

```diff
@@ -342,7 +411,7 @@ func (s *testEncryptSuite) TestAESDecryptWithCBC(c *C) {
 	}
 }
 
-func (s *testEncryptSuite) TestAESEncryptWithOFB(c *C) {
+func (s *testEncryptSuite) AESEncryptWithCFB(c *C) {
```

Comment:
s/ AESEncryptWithCFB/ TestAESEncryptWithCFB ?

Reply:
The master branch had a test name mismatch which I had tried to resolve
https://github.com/pingcap/tidb/blob/master/util/encrypt/aes_test.go#L345
Test Name Reflects at `OFB` mode is being tested, whereas the test if for `AESEncryptWithCFB`